exercises/3-operations-dr-replication/ReadMe.md

```diff
@@ -1,20 +1,28 @@
 Enable Disaster Recovery Replication
 ===
 In this exercise, you will enable disaster recovery replication between the primary and secondary vault clusters.
-Read the [Disaster Recovery Replication](https://www.vaultproject.io/docs/enterprise/disaster-recovery) documentation.
+Read the [Disaster Recovery Replication](https://developer.hashicorp.com/vault/tutorials/enterprise/disaster-recovery) documentation.
 Use the vault ui to enable disaster recovery replication.
 The DR vault is already deployed on the "vault-dr" namespace and initialized and unsealed. 
-You can find the root token and unseal keys  in the cluster-keys-dr.json file.
+You can find the root token and unseal keys  in the `cluster-keys-dr.json` file.
 
 ## Setup DR replication
-Go to https://vault-primary.vault-boost and login with the root token.
+Go to https://vault-primary.vault-boost.lab and login with the root token.
 Setup the vault-primary cluster as the dr primary.
-Then go to https://vault-dr.vault-boost and login with the root token and setup the vault-dr cluster as the dr secondary.
+Then go to https://vault-dr.vault-boost.lab and login with the root token and setup the vault-dr cluster as the dr secondary.
 
 Hint: 
-    - cluster address: https://vault-primary-active.vault-primary.svc.cluster.local:8201
-    - cluster primary api address: https://vault-primary-active.vault-primary.svc.cluster.local:8200
-    - ca file: /vault/tls/vault.ca
+- cluster address: https://vault-primary-active.vault-primary.svc.cluster.local:8201
+- cluster primary api address: https://vault-primary-active.vault-primary.svc.cluster.local:8200
+- ca file: /vault/tls/vault.ca
 
-Don't forget to unseal the DR vault again.
```

Comment:
As I observed the cluster will just switch to secondary mode automatically and there is no need to unseal anything. However, there is only one Pod in the cluster which receives the replication streams (warm-standby cluster). This is what I added to the next section as explanation.